command/help/_help_generator.ts

```diff
@@ -358,7 +358,7 @@ export class HelpGenerator {
 }
 
 function capitalize(string: string): string {
-  return string?.charAt(0).toUpperCase() + string.slice(1) ?? "";
+  return string?.charAt(0).toUpperCase() + string.slice(1);
```

Comment:
The optional chaining can be also removed.
```suggestion
  return string.charAt(0).toUpperCase() + string.slice(1);
```